src/components/RequestgroupsList.vue

```diff
@@ -425,24 +424,40 @@ export default {
         this.rgQueryParams.user = this.profile.username;
       }
     },
+    clearErrors: function() {
+      for (let error of this.rgQueryErrors) {
```

Comment:
Yep they are displayed in a different component. Messages that get placed in the store are displayed on `App.vue`, which is the component that provides consistent framing for the pages - header, footer, page alerts. So, placing these messages in the store makes them pop up in that location, which is right about where I wanted them displayed anyway. I figured since I added the mechanism to display alerts consistently in `App.vue` that I would switch to using it for the messages generated by `RequestgroupsList.vue`.